mainapp/migrations/0012_contributors.py

```diff
@@ -19,6 +19,7 @@ class Migration(migrations.Migration):
                 ('phone', models.CharField(max_length=10)),
                 ('address', models.TextField()),
                 ('commodities', models.TextField(verbose_name='What you can contribute')),
+                ('blood_group', models.CharField(max_length=10)),
```

Comment:
You're modifying an existing migration file - this will have no effect when running the migrations (and may break things for testers). You need to create (auto-generate) a new migration file.

Reply:
Documentation: https://docs.djangoproject.com/en/2.1/topics/migrations/

Reply:
@bhumilharia +1 
[Django migration files should not be changed manually as it may lead to cyclic dependencies](https://stackoverflow.com/a/37097258). Running this on existing (0012+) based systems will not have an effect as django stores the migrations which it has already applied to the db in `django_migrations` table 

Ideally, the column should be added in the `models.py`, and as reccommeded by you, the migration file be auto generated using `python manage.py makemigrations` to be then migrated as `python manage.py migrate`

Adding to all of this, I feel, the `blood_group` should **NOT** be a open char field. It should use [Django choices](https://docs.djangoproject.com/en/2.1/ref/models/fields/#choices) to enforce data integrity and ease of use

Reply:
@darth-dodo  thanks for the suggestion , sounds fine. Will implement it.